documentation/docs/user_guide/inputs/meteorological_forcing.md

```diff
@@ -0,0 +1,124 @@
+# Meteorological forcing for offline CABLE only
+
+The meteorological forcing is a collection of meteorological variables that need to be read into CABLE's `met` arrays when CABLE is run in offline (uncoupled) mode.
+
+The filename is set using the CABLE namelist variable `filename%met`.
+The global offline simulations use multiple met files, one for each meteorological variable recorded in compressed grid method.
+The data must be in NetCDF format with the correct units, broadly conforming to the [ALMA](https://web.lmd.jussieu.fr/~polcher/ALMA/convention_output_3.html) standard.
+
+An example input file can can be found at the [NCI THREDDS server](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f7075_4625_2374_0846).
+
```

Comment:
```suggestion
!!! note "Number of land points in the simulation"

    The number of points simulated by offline CABLE is not specified in the code but is determined using the number of land points found in the meteorological forcing file. CABLE will consider the number of patches to determine the number of simulated points according to the [`patchfrac` variable](#optional-variables).
```

I suggest moving this here instead of burying it after the grid specifications.